### ISSUE TITLE
Issues when using NgxPopperModule.forRoot() with AoT bulid option

### ISSUE THREAD

**Issue by slack4u** [MrFrankel/ngx-popper#27]
I'm  getting the following error message when I use the command `ng build --aot` or `ng build --prod`.

```
ERROR in Error during template compile of 'AppModule'
  Function calls are not supported in decorators but 'NgxPopperModule' was called.
```

Environment: 
```
Angular CLI: 1.6.3
Node: 8.9.1
OS: darwin x64
Angular: 5.2.0
... animations, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... router

@angular/cli: 1.6.3
@angular-devkit/build-optimizer: 0.0.36
@angular-devkit/core: 0.0.22
@angular-devkit/schematics: 0.0.42
@ngtools/json-schema: 1.1.0
@ngtools/webpack: 1.9.3
@schematics/angular: 0.1.11
@schematics/schematics: 0.0.11
typescript: 2.6.2
webpack: 3.10.0
```

**Comment by MrFrankel** [comment]
@slack4u seems like a knows issue in angular, i'm working on a fix.
For now you can revert to directly changing the static defaults like so:
`PopperController.baseOptions.disableAnimation = true;`

Ill update once this is resolved, thanks for bringing this up.

**Comment by MrFrankel** [comment]
@slack4u Issue should be fixed in v1.9.5.

**Comment by slack4u** [comment]
@MrFrankel It works great. Thanks!
